src/Datadog.Trace/Datadog.Trace.csproj

```
@@ -30,4 +30,31 @@
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
   </ItemGroup>
 
+  <!-- Below this point is reserved for vendor items -->
+  <!--Serilog-->
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net45' ">
+    <DefineConstants>REMOTING;HASHTABLE</DefineConstants>
```

We should append to any previously defined constants instead of overwriting them:
```suggestion
    <DefineConstants>$(DefineConstants);REMOTING;HASHTABLE</DefineConstants>
```